.travis.yml

```diff
@@ -1,7 +1,15 @@
 language: python
-python:
-   - "2.7"
-   - "3.4"
+jobs:
+  include:
+    - name: "Python 2.7 on Xenial"
+      python: "2.7"
+      dist: "xenial"
```

Comment:
Do we need to specify `dist`? What about the old approach to test on specified python versions?

Reply:
Xenial is the default, but next time we revisit this library in 10 years, I guess, so I decided to hardcode the exact version to avoid surprises.